reactDashboard/src/App.jsx

```diff
@@ -7,27 +7,53 @@ import DataSources from "./pages/DataSources";
 import Overview from "./pages/Overview";
 import Regional from "./pages/Regional";
 import {
-  PAGE_ABOUT_US,
-  PAGE_ACCESSIBILITY,
-  PAGE_DATA_SOURCES,
-  PAGE_OVERVIEW,
-  PAGE_REGIONAL,
+  URL_ABOUT_US,
+  URL_ACCESSIBILITY,
+  URL_DATA_SOURCES,
+  URL_OVERVIEW,
+  URL_REGIONAL,
 } from "./pages/PageConsts";
 import Footer from "./components/Footer/Footer";
 import TagManager from "react-gtm-module";
 import { readCsvData, fetchAndStore } from "./components/Utils/CsvUtils";
 import { stopAudio } from "./components/Utils/Sonification";
 import DashboardNavbar from "./components/DashboardNavbar/DashboardNavbar";
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Redirect,
+  useLocation,
+} from "react-router-dom";
 
 const tagManagerArgs = {
   gtmId: "GTM-5LKHW33",
 };
 
 TagManager.initialize(tagManagerArgs);
 
-const App = () => {
-  const [currentPage, setCurrentPage] = useState(PAGE_OVERVIEW);
+const ScrollToTop = () => {
```

Comment:
These need to be components to use hooks (useLocation) and still be inside the <Router> hierarchy